modsecurity/README.md

```diff
@@ -0,0 +1,126 @@
+JavaCPP Presets for ModSecurity
+================================
+
+[![Gitter](https://badges.gitter.im/bytedeco/javacpp.svg)](https://gitter.im/bytedeco/javacpp) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.bytedeco/modsecurity/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.bytedeco/modsecurity) [![Sonatype Nexus (Snapshots)](https://img.shields.io/nexus/s/https/oss.sonatype.org/org.bytedeco/modsecurity.svg)](http://bytedeco.org/builds/)  
+<sup>Build status for all platforms:</sup> [![modsecurity](https://github.com/bytedeco/javacpp-presets/workflows/modsecurity/badge.svg)](https://github.com/bytedeco/javacpp-presets/actions?query=workflow%3Acpu_features)  <sup>Commercial support:</sup> [![xscode](https://img.shields.io/badge/Available%20on-xs%3Acode-blue?style=?style=plastic&logo=appveyor&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAMAAACdt4HsAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////////VXz1bAAAAAJ0Uk5T/wDltzBKAAAAlUlEQVR42uzXSwqAMAwE0Mn9L+3Ggtgkk35QwcnSJo9S+yGwM9DCooCbgn4YrJ4CIPUcQF7/XSBbx2TEz4sAZ2q1RAECBAiYBlCtvwN+KiYAlG7UDGj59MViT9hOwEqAhYCtAsUZvL6I6W8c2wcbd+LIWSCHSTeSAAECngN4xxIDSK9f4B9t377Wd7H5Nt7/Xz8eAgwAvesLRjYYPuUAAAAASUVORK5CYII=)](https://xscode.com/bytedeco/javacpp-presets)
+
+
+Introduction
+------------
+This directory contains the JavaCPP Presets module for:
+
+* ModSecurity 3.0.4  https://github.com/SpiderLabs/ModSecurity
+
+Please refer to the parent README.md file for more detailed information about the JavaCPP Presets.
+
+
+Documentation
+-------------
+Java API documentation is available here:
+
+* http://bytedeco.org/javacpp-presets/modsecurity/apidocs/
+
+Build Notes
+------------
+
+To Build JavaCPP ModSecurity Preset libraries requred by ModSecurity should be installed. 
+
+Detailed information can be found here:
+
+* https://github.com/SpiderLabs/ModSecurity/wiki/Compilation-recipes-for-v3.x
+
+Sample Usage
+------------
+Here is a simple example of ModSecurity ported to Java from this C source file:
+
+* https://github.com/SpiderLabs/ModSecurity#simple-example-using-c
+
+We can use [Maven 3](http://maven.apache.org/) to download and install automatically all the class files as well as the native binaries. To run this sample code, after creating the `pom.xml` and `ModSecuritySimpleIntervention.java.java` source files below, simply execute on the command line:
+```bash
+ $ mvn compile exec:java
+```
+
+### The `pom.xml` build file
+```xml
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.bytedeco.modsecurity</groupId>
+  <artifactId>modsecurity-sample</artifactId>
+  <version>1.5.5-SNAPSHOT</version>
+  <properties>
+    <exec.mainClass>ModSecuritySimpleIntervention</exec.mainClass>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>org.bytedeco</groupId>
+      <artifactId>modsecurity-platform</artifactId>
+      <version>3.0.4-1.5.5-SNAPSHOT</version>
```

Comment:
The versions haven't been updated in the README.md file.

Reply:
I still see 3.0.4 in the README.md...

Reply:
maybe you did not update? I fixed it with the last commit today.

Reply:
No, still seeing it in the update: https://github.com/bytedeco/javacpp-presets/blob/045c7f5e114509fd442e0a34f5f5287bb33daa24/modsecurity/README.md#introduction

Reply:
Sorry, I focused on commented lines. Now it`s already fixed. Thanks!